caringcaribou/modules/uds.py

```diff
@@ -169,7 +169,7 @@ def is_valid_response(message):
 
     found_arbitration_ids = []
 
-    with IsoTp(None, None) as tp:
+    with IsoTp(None, None, padding_value=None) as tp:
```

Comment:
The ISO-TP support for padding (implemented in 3713cec2898f7373211fc517e04d87bbbdefd06b) should be used here, rather than turning it off and reinventing the ISO-TP padding on a higher protocol level inside the UDS module :slightly_smiling_face: